BedrockServer.cpp

```diff
@@ -1425,10 +1407,16 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
         }
     }
 
-    // If we started shutting down, we can now finish that process.
+    // If we've been told to start down, we'll set the lastChance timer.
```

Comment:
this comment is a little weird now `If we've been told to start down` maybe `If we've been told to start shutdown`